.github/workflows/generate_workflows_lib/src/generate_workflows_lib/core_contrib_test.yml.j2

```diff
@@ -27,12 +28,6 @@ jobs:
           repository: open-telemetry/opentelemetry-python-contrib
           ref: ${% raw %}{{ env.CONTRIB_REPO_SHA }}{% endraw %}
 
-      - name: Checkout core repo @ SHA - ${% raw %}{{ github.sha }}{% endraw %}
```

Comment:
I'm not 100% sure but I think this has no use since we install packages in tox pointing to git+url@CORE_REPO_SHA

Reply:
It should be fine, grepping the checkout dir `opentelemetry-python-core` returns nothing other than this